Add October 2023 newsletter (#2037)
writethedocs/www: docs/blog/newsletter-october-2023.rst

.. post:: October 06, 2023
  :tags: newsletter

#########################################
Write the Docs Newsletter – October 2023
#########################################

Ave, documentarians. Halloween may be approaching, but there's nothing scary about this month's batch of stories from the world of Write the Docs.

The Write the Docs `Documentarian Salary Survey for 2023 <https://salary-survey.writethedocs.org/>`__ is open and accepting submissions! This is the survey's fifth year and, in addition to our regular salary and job-related questions, we're exploring our community's thoughts on "back to the office" mandates, feelings about job security in a volatile market, and attitudes towards pay transparency.

So whether you're an employee, contractor, freelancer, or currently unemployed, `head on over <https://salary-survey.writethedocs.org/>`_ and help us make this year's survey the most useful yet.

> In conference news, `tickets for the in-person Australia conference </conf/australia/2023/tickets/>`__ are still open. If you'd like to apply for an `opportunity grant </conf/australia/2023/opportunity-grants/>`__, you have until 15 October.

Reviewer: Speakers should be announced soon, right @swapnilogale ?

Reviewer 2: Yes, just overnight if I can

> In conference news, `tickets for the in-person Australia conference </conf/australia/2023/tickets/>`__ are still open. If you'd like to apply for an `opportunity grant </conf/australia/2023/opportunity-grants/>`__, you have until 15 October.

Author: ```suggestion
In conference news, the in-person Australia conference has just `announced its speakers </conf/australia/2023/news/announcing-speakers/>. Check out the amazing talks you have to look forward to and then `grab a ticket </conf/australia/2023/tickets/>`__. If you'd like to apply for an `opportunity grant </conf/australia/2023/opportunity-grants/>`__, you have until 15 October.
```

> In conference news, the in-person Australia conference has just `announced its speakers </conf/australia/2023/news/announcing-speakers/>. Check out the amazing talks you have to look forward to and then `grab a ticket </conf/australia/2023/tickets/>`__. If you'd like to apply for an `opportunity grant </conf/australia/2023/opportunity-grants/>`__, you have until 15 October.

Author: ```suggestion
In conference news, the in-person Australia conference has just `announced its speakers </conf/australia/2023/news/announcing-speakers/>`__. Check out the amazing talks you have to look forward to and then `grab a ticket </conf/australia/2023/tickets/>`__. If you'd like to apply for an `opportunity grant </conf/australia/2023/opportunity-grants/>`__, you have until 15 October.
```

- Some documentation systems focus on short, succinct topics. This may require many links to cross-reference content adequately.

A related issue concerns *where* to put the links: throughout the content or at the end of the content in a separate list? The main arguments against putting links at the end: the link may have lost its context or the reader may never scroll that far. In either case, it’s important to use relevant text. Having the link within the content may disrupt the reader by triggering cognitive overload and making them wonder: "Should I click or not? Should I click as I come to the link or wait until I finish reading?"

If you put links within the content, `one reference suggested <https://readabilityguidelines.co.uk/content-design/links/#2-avoid-mid-sentence-links>`__ always putting the link at the end of the sentence to minimize distraction. Accessibility was an additional concern, but exact solutions were unclear because interpretation depends upon the screen reader. 

--------------------------------
Personality in Technical Writing
--------------------------------

> There are still different views among documentarians on the issue of the importance of personality, customization, and tone in technical writing. People recently shared their thoughts on the topic, giving different points of view on whether such things belong in technical documents.

Reviewer: ```suggestion
There are still different views among documentarians on the issue of the importance of personality, customization, and tone in technical writing.
```

harsh, but...

> Some people said that strict accuracy was the most important in technical writing, while others said that there was room for creativity and humor as long as precision wasn’t compromised. In particular, they said that adding a bit of personality can make such writing more interesting, especially if the readers can handle it.

Reviewer 3: "the most important in technical writing" seems like it needs a noun, like "the most important consideration in technical writing"

Reviewer: @hillaryfraley  Good call! Alternatively "accuracy was most important in ..."

Author: I remember changing it to that and thinking, "Remember to come back and add a noun." 🤦 Thanks!

Author: ```suggestion
Some people said that strict accuracy was the most important consideration in technical writing, while others said that there was room for creativity and humor as long as precision wasn’t compromised. In particular, they said that adding a bit of personality can make such writing more interesting, especially if the readers can handle it.
```

-----------------------------------

Someone who switched jobs recently asked in `#docs-as-code <https://writethedocs.slack.com/archives/C72NZ18FR>`__ about the benefits and drawbacks of keeping docs in the same repository as app code. They moved from having docs in a separate repository to docs in the same repository as the product source code and were feeling frustrated.

Some of the perceived drawbacks to keeping docs with app code included changes feeling slower and less flexible, requiring PRs for everything, even small typos. Each change also required reviews from developers and lots of linting when app code wasn't touched. Also, collaboration seemed more difficult than in tools like Google Docs and Notion. In short, it felt like there were roadblocks everywhere.

People noted some potential benefits, including increased accountability for changes with PRs. Some also felt safer knowing any mistakes weren't theirs alone and that the CI checks and reviews helped keep entire pages from breaking. Someone also noted that with docs in the same repository as code, it was easier to enforce docs changes when code is updated, including potentially automating some changes.

> People had some suggestions to help overcome some roadblocks. Opening PRs early in the process lets reviewers check out changes locally to see the effects. Someone also suggested using the principle of pair programming in pair reviews for easier collaboration: run the docs locally and walk through the changes on a video call. Another idea was keeping PRs small and therefore manageable, but batching really small things like typos together if each PR takes too long. And encouraging improvements to the CI to catch typos earlier and also only runing when needed (not linting untouched code).

Reviewer 3: ```suggestion
People had some suggestions to help overcome some roadblocks. Opening PRs early in the process lets reviewers check out changes locally to see the effects. Someone also suggested using the principle of pair programming in pair reviews for easier collaboration: run the docs locally and walk through the changes on a video call. Another idea was keeping PRs small and therefore manageable, but batching really small things like typos together if each PR takes too long. And encouraging improvements to the CI to catch typos earlier and also only running when needed (not linting untouched code).
```

Reviewer: Sad to see no rune emoji

Author: That's because it's already a part of Unicode (ᚠ,ᛃ,ᚬ): https://en.wikipedia.org/wiki/Runic_(Unicode_block)

> People had some suggestions to help overcome some roadblocks. Opening PRs early in the process lets reviewers check out changes locally to see the effects. Someone also suggested using the principle of pair programming in pair reviews for easier collaboration: run the docs locally and walk through the changes on a video call. It was also suggested to keep PRs small and therefore manageable, but batch really small things like typos together if each PR takes too long. And encourage improvements to the CI to catch typos earlier and also only run when needed (not linting untouched code).

Reviewer: Random passive for no reason here?

Author: I think the idea was to have variety in sentence intros, but can change it.

Author: ```suggestion
People had some suggestions to help overcome some roadblocks. Opening PRs early in the process lets reviewers check out changes locally to see the effects. Someone also suggested using the principle of pair programming in pair reviews for easier collaboration: run the docs locally and walk through the changes on a video call. Another idea was keeping PRs small and therefore manageable, but batching really small things like typos together if each PR takes too long. And encouraging improvements to the CI to catch typos earlier and also only runing when needed (not linting untouched code).
```
